### PR TITLE
added a piechart in profileactivity 

### LINKED ISSUE #79: Display pie chart of the count of different lists in `ProfileActivity`
### Suggestion / Feature Request

Display a pie chart in the `ProfileActivity` showing the count of movies or tv shows with the  status
- Watching 
- Completed
- On Hold
- Dropped
- Plan To Watch

Use the corresponding colors of each list when representing them in pie chart

### PR DESCRIPTION
**Pull request by armaanbadhan**
## Description

pie chart added which shows counts of different lists of user

## Issue Resolved

fixes #79 

## Screenshots 

<div>

</div>
## Checklist

Please make sure to review the following before submitting your PR: 

- [x] I have read the [contribution guidelines](https://github.com/PEC-CSS/MovieDroid/blob/main/CONTRIBUTING.md).
- [x] I have read the code of conduct.
- [x] I have reviewed my submission in detail.

### CODE REVIEW

File: app/src/main/res/layout/activity_profile.xml [hunk 1]
```diff
@@ -21,23 +21,22 @@
...
     <LinearLayout
-        android:id="@+id/linearLayout"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:gravity="center"
-        android:orientation="vertical"
-        app:layout_constraintBottom_toBottomOf="parent"
+        android:id="@+id/linearLayout3"
+        android:layout_width="match_parent"
+        android:layout_height="135dp"
```
Comment (ken1000minus7) [inline app/src/main/res/layout/activity_profile.xml]: Use `sdp` instead of `dp` eberywhere

File: app/src/main/res/layout/activity_profile.xml [hunk 2]
```diff
@@ -21,23 +21,22 @@
...
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
```
Comment (ken1000minus7) [inline app/src/main/res/layout/activity_profile.xml]: Here as well

File: app/src/main/res/layout/activity_profile.xml [hunk 3]
```diff
@@ -21,23 +21,22 @@
...
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.35">
+        app:layout_constraintTop_toBottomOf="@+id/back_btn_profile">
 
-        <!--     google profile pic-->
         <androidx.cardview.widget.CardView
-            android:layout_width="@dimen/_100sdp"
-            android:layout_height="@dimen/_100sdp"
+            android:layout_width="135dp"
```
Comment (ken1000minus7) [inline app/src/main/res/layout/activity_profile.xml]: Here too

File: app/src/main/res/layout/activity_profile.xml [hunk 4]
```diff
@@ -46,45 +45,71 @@
...
+        </LinearLayout>
 
     </LinearLayout>
 
+    <View
+        android:id="@+id/divider2"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_marginStart="1dp"
+        android:layout_marginEnd="1dp"
```
Comment (ken1000minus7) [inline app/src/main/res/layout/activity_profile.xml]: aaaand here

File: app/src/main/res/layout/activity_profile.xml [hunk 5]
```diff
@@ -96,15 +121,41 @@
...
         </LinearLayout>
+
     </LinearLayout>
 
+    <View
+        android:id="@+id/divider3"
+        android:layout_width="409dp"
+        android:layout_height="1dp"
+        android:layout_marginStart="1dp"
+        android:layout_marginEnd="1dp"
```
Comment (ken1000minus7) [inline app/src/main/res/layout/activity_profile.xml]: here here too too

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]
Looks nice, gw,, small changes required though

**Comment by armaanbadhan** [comment]
done!

**Review by ken1000minus7 — APPROVED** [review]
Lgtm gw

**Pull request merged**